core/modules/cart/store/actions/syncrhonizeActions.ts

```diff
@@ -70,7 +73,31 @@ const synchronizeActions = {
     Logger.error(result, 'cart')
     cartHooksExecutors.afterSync(result)
     return createDiffLog()
-  }
+  },
+  stockSync: debounce(async ({ dispatch, commit }, stockTask) => {
```

Comment:
You added denounce for a case that cart sync is executed in short intervals and stock endpoint responding slowly? Or what’s the case for this?

Reply:
This is old action `stockAfterCheck` which was moved to the cart module. I used `debounce` instead of `setTimeout`.

Reply:
I've been testing that - doesn't work without that timeout and I don't really know why, hard to explain, it seems like there is no `result` field (even thought i can see it in the console.log) but it appears afters some time... that's why probably someone used a delay here